drivers/usb/device/usb_dc_dw.c

```diff
@@ -71,48 +71,40 @@ static inline void _usb_dw_int_unmask(void)
 #endif
 }
 
-#if (CONFIG_SYS_LOG_USB_DRIVER_LEVEL >= SYS_LOG_LEVEL_DEBUG)
+#if (LOG_LEVEL >= LOG_LEVEL_DEBUG)
 static void usb_dw_reg_dump(void)
 {
 	u8_t i;
 
-	SYS_LOG_DBG("USB registers:");
-	SYS_LOG_DBG("  GOTGCTL : 0x%x", USB_DW->gotgctl);
-	SYS_LOG_DBG("  GOTGINT : 0x%x", USB_DW->gotgint);
-	SYS_LOG_DBG("  GAHBCFG : 0x%x", USB_DW->gahbcfg);
-	SYS_LOG_DBG("  GUSBCFG : 0x%x", USB_DW->gusbcfg);
-	SYS_LOG_DBG("  GINTSTS : 0x%x", USB_DW->gintsts);
-	SYS_LOG_DBG("  GINTMSK : 0x%x", USB_DW->gintmsk);
-	SYS_LOG_DBG("  DCFG    : 0x%x", USB_DW->dcfg);
-	SYS_LOG_DBG("  DCTL    : 0x%x", USB_DW->dctl);
-	SYS_LOG_DBG("  DSTS    : 0x%x", USB_DW->dsts);
-	SYS_LOG_DBG("  DIEPMSK : 0x%x", USB_DW->diepmsk);
-	SYS_LOG_DBG("  DOEPMSK : 0x%x", USB_DW->doepmsk);
-	SYS_LOG_DBG("  DAINT   : 0x%x", USB_DW->daint);
-	SYS_LOG_DBG("  DAINTMSK: 0x%x", USB_DW->daintmsk);
-	SYS_LOG_DBG("  GHWCFG1 : 0x%x", USB_DW->ghwcfg1);
-	SYS_LOG_DBG("  GHWCFG2 : 0x%x", USB_DW->ghwcfg2);
-	SYS_LOG_DBG("  GHWCFG3 : 0x%x", USB_DW->ghwcfg3);
-	SYS_LOG_DBG("  GHWCFG4 : 0x%x", USB_DW->ghwcfg4);
+	USB_DBG("USB registers:");
+	USB_DBG("  GOTGCTL : 0x%x", USB_DW->gotgctl);
```

Comment:
I would recommend to rephrase those messages to use more parameters. Internally, log messages takes 32 bytes of ram for 0-3 arguments and 64 bytes for 4-9. So it can be grouped in threes for better ram utilization.

Reply:
Done